examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -15,9 +15,16 @@ import {INITIAL_EXAMPLE_NAME, EXAMPLES} from './examples';
 import DebugPanel from './components/debug-panel';
 import ControlPanel from './components/control-panel';
 
-import {INITIAL_MAP_STYLE, CONTRAST_MAP_STYLES, INITIAL_COLORING_MODE} from './constants';
+import {
+  INITIAL_MAP_STYLE,
+  CONTRAST_MAP_STYLES,
+  INITIAL_TILE_COLORING_MODE,
+  INITIAL_OBB_COLORING_MODE
+} from './constants';
 import {getFrustumBounds} from './frustum-utils';
 import TileLayer from './tile-layer/tile-layer';
+import ObbLayer from './obb-utils';
+import ColorMap from './coloring-utils';
```

Comment:
color-utils?

Reply:
Renamed to color-map.js

Reply:
And obb-utils.js renamed to obb-layer.js